### PR TITLE
update readme file

### PR DESCRIPTION
**Pull request by DanNiESh**
add flocx-market service creation, installation and configuration and instructions to run the services.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,97 @@
-# FLOCX Marketplace
-Just a simple Flask application for FLOCX Marketplace project.
-[More details on FLOCX](https://github.com/CCI-MOC/flocx)
+# flocx-market
```
Comment (tzumainn) [inline README.md]: Let's keep the title the same - 'FLOCX Marketplace'

Comment (DanNiESh) [reply README.md]: Okay.

**Comment by DanNiESh** [comment]
CI build failed: commit message does not reference related issue/task/user story
There's no user story related to updating flocx-market instructions.

**Comment by DanNiESh** [comment]
> Hi! This looks good - I'm just running through the instructions to confirm. We may want to take out the usage of `sudo` - it's not necessary if you're using a venv (and I was probably wrong to specify using sudo in esi-leap).

Ah, flocx-market-dbsync create_schema is not working because there's no such function inside dbsync.py. I'll fix it.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,97 @@
...
+
+```
+openstack user create flocx-market --password PASSWORD
+```
+
+Assign the admin role to the user-project pair:
+
+```
+openstack role add --project service --user flocx-market admin
+```
```
Comment (tzumainn) [inline README.md]: Let's move this entire section running openstack commands to the end - after instructions on how to start up the API.

**Comment by DanNiESh** [comment]
All fixed. Seems like the commit should include a Taiga task in order to pass CI test.

**Comment by tzumainn** [comment]
Or a github issue! I've created one: https://github.com/CCI-MOC/flocx-market/issues/94

Just add `#94` to your commit message.

**Pull request merged**